### PR TITLE
Asutula/ios snippets

### PR DESCRIPTION
**Pull request by asutula**

### CODE REVIEW

**Comment by asutula** [comment]
Ok, rebased and those conflicts are gone. I'm sure there are still some holes in this, but my intention was to fill out all the objc and swift items possible. G2G.

File: docs/a-tour-of-textile.md [hunk 1]
```diff
@@ -1349,6 +1453,10 @@ Take a closer look at the "My runs" thread using the `blocks ls` command, which
 {{examples.my_runs.blocks_ls.objc.code}}
 ```
 
+```Swift tab="Swift"
+{{examples.my_runs.blocks_ls.swift.code}}
```
Comment (andrewxhill) [inline docs/a-tour-of-textile.md]: even though it's a todo, this needs to be added as an object in the YAML or it wont compile

Comment (asutula) [reply docs/a-tour-of-textile.md]: Ah yea I meant to add things to the yml where they were missing, but I'm sure I missed some.

Comment (asutula) [reply docs/a-tour-of-textile.md]: Fixed.

**Pull request merged**